client/src/Routes.jsx

```diff
@@ -31,21 +34,36 @@ export default class Routes extends Component {
             exact path='/'
             component={Home}
           />
+          
+        </Route>   
+        <Route exact path='/nba'>
           <Switch>
-            <div className='most-wrapper'>
-              <div className='most-news-container'>
-                <MostPopularComments
-                  blockTitle='most popular'
-                  newsBlock={MostPopular}
-                />
-                <MostPopularComments
-                  blockTitle='most comments'
-                  newsBlock={MostComments}
-                />
-              </div>
-            </div>
+            <Header />
+          </Switch>
+          <Switch>
+            <Menu menuList={menuData.menu} />
```

Comment:
Why we wrapp in Switch?

Reply:
we need it to only use it on my page

Reply:
Menu is only on your page? What about home page?